imports/ui_2/modals/autoform-edit.html

```diff
@@ -14,7 +14,7 @@ <h4 class="modal-title" id="myModalLabel">{{_ title}}</h4>
           <div class="modal-footer">
             <div>
               <button type="button" class="btn btn-default" data-dismiss="modal">{{_ "cancel"}}</button>
-              <button type="submit" class="btn btn-primary">{{_ "save"}}</button>
+              <button type="submit" class="btn btn-primary {{#unless userHasPermission 'parcels.update' communityId}}disabled{{/unless}}">{{_ "save"}}</button>
```

Comment:
Ez igy most csak a parcels.update-es szerkeszto modalban mukodik helyesen.
Van meg sok mas autoform-unk is azon kivul.

Reply:
Hát igen, ez teljesen jogos :) A funkció másolása közben megfeledkeztem arról, hogy nem csak ez az egy felület hívja meg, hanem másik ezer is.